src/ClusterBootstrap/storage/glusterFS/gk-deploy

```diff
@@ -207,7 +207,7 @@ check_pods() {
       break
     fi
     sleep 2
-    pods=$(${CLI} get pod --no-headers --show-all --selector="${1}" 2>&1)
+    pods=$(${CLI} get pod --no-headers --selector="${1}" 2>&1)
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.10.md#deprecations

> --show-all (which only affected pods and only for human readable/non-API printers) is now defaulted to true and deprecated. It will be inert in 1.11 and removed in a future release.